integrationtests/cli/assets/helmrepository/testrepo/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
This seems to be the default chart template from helm. It doesn't seem like the tests need this. Except for the file check against 'svc.yaml' and 'deployment.yaml'.

Can we use/copy a smaller chart, e.g. the "config-chart" from https://github.com/rancher/fleet-examples/commit/f1be9e1bd0387fb6ec0df35f38b147a7016937e6 ?